cmd/helm/install.go

```diff
@@ -94,7 +94,10 @@ And in the following example, 'foo' is set to '{"key1":"value1","key2":"bar"}':
     $ helm install --set-json='foo={"key1":"value1","key2":"value2"}' --set-json='foo.key2="bar"' myredis ./redis
 
 To check the generated manifests of a release without installing the chart,
-the '--debug' and '--dry-run' flags can be combined.
+the --debug and --dry-run flags can be combined.
+
+The --dry-run flag will output all generated chart manifests, including Secrets.
```

Comment:
```suggestion
The --dry-run flag will output all generated chart manifests, including Secrets which can be decoded exposing the underlying values.
```

Reply:
Because these secrets are chart/template generated they may be plain text. This is before they are sent to Kubernetes.

Reply:
Update Updated proposed changes